website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -19,12 +19,17 @@ function _downloadEvent(event, item, col) {
 }
 
 // Define Fangorn Button Actions
-function _fangornActionColumn (item, col) {
-    var self = this;
+function _dataverseDefineToolbar(item) {
```

Comment:
I'm thinking of ways to achieve this but I'm not sure how to implement it across addons, let's talk briefly about this in the morning.